codetable/src/lib.rs

```diff
@@ -36,11 +59,11 @@ pub enum Code {
     #[mh(code = 0x14, hasher = crate::Sha3_512)]
     Sha3_512,
     /// Keccak-224 (28-byte hash size)
-    #[cfg(feature = "sha3")]
+    #[cfg(feature = "sha2")]
```

Comment:
The features on the table from here to the bottom should match the original ones. There shouldn't be any difference.

Reply:
Yep, you are right. Sorry for the sloppyness.